integration/network/service_test.go

```diff
@@ -49,14 +48,12 @@ func TestServiceWithPredefinedNetwork(t *testing.T) {
 	require.NoError(t, err)
 
 	poll.WaitOn(t, serviceIsRemoved(client, serviceID), pollSettings)
-	poll.WaitOn(t, noTasks(client), pollSettings)
+	poll.WaitOn(t, noServices(client), pollSettings)
 
 }
 
 func serviceRunningCount(client client.ServiceAPIClient, serviceID string, instances uint64) func(log poll.LogT) poll.Result {
 	return func(log poll.LogT) poll.Result {
-		filter := filters.NewArgs()
-		filter.Add("service", serviceID)
```

Comment:
Removing this is not the correct fix. It should be used as a filter in `ServiceList`.

Reply:
1) The actual regression issue was we couldnt create service with host network. Thats what I tried to test it to make sure there is no issue in creating service with host network. 
2) For your clean up comment :  I try to remove service and make sure service is getting cleared. Thats what  I meant by cleanup. I am not doing anything extra. Do you suggest I dont need to do verify the service that I created  whether I am able to remove or not?. Pls let me know your feedback on this.
3)Regarding not adding filter: I am trying to get all the services running. so looked into existing examples of functions calling serviceList API. none of them using filter. Your previous comment on original comment said , I have defined filter but not using it. So i wanted to clean up the code as part of your previous comment. Pls correct me if my understand is wrong.

Reply:
>  I try to remove service and make sure service is getting cleared. Thats what I meant by cleanup. I am not doing anything extra. Do you suggest I dont need to do verify the service that I created whether I am able to remove or not?. Pls let me know your feedback on this.

That's right. The daemon which runs this service is going to be shutdown when the test exits (`defer d.Stop(t)`) so you don't need to spend time doing cleanup.

> Regarding not adding filter: I am trying to get all the services running.

Ok, I guess this is ok since you're targeting a daemon that was launched for this test.